float32[]   rho     # array of rho values
float32[]   theta   # array of angles
float32[]   q       # point along line perpendicular to the line given by rho-theta